packages/assistify-help-request/client/views/creationDialog/AssistifyWordCloud.js

```diff
@@ -0,0 +1,32 @@
+import {ReactiveVar} from 'meteor/reactive-var';
+
+const WordCloud= require('meteor/overture8:wordcloud2');
+
+function drawWords() {
+	const instance = Template.instance();
+	const properties = instance.data.properties;
+	//properties.setCanvas(instance.canvasToDraw.get());
+	window.WordCloud(instance.canvasToDraw.get(), properties);
+}
+
+Template.AssistifyWordCloud.events({
```

Comment:
if we don't need them, we don't need them - but delete them ;)

Reply:
@mrsimpson Ja, I will need it soon or later to implement a word highlighting.